terraform/README.md

```diff
@@ -166,28 +168,22 @@ However, ghaf-infra resources can be deployed to other Azure locations too, with
 - For a full list of available Azure location names, run `az account list-locations -o table` in ghaf-infra devshell.
 - Not all Azure VM sizes or other resources are available in all Azure locations. You can search the availability of specific resources through the Azure region product page e.g.: https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?regions=europe-north&products=virtual-machines. Alternatively, you can list the VM sizes per location with `az vm list-sizes` command from the ghaf-infra devshell, for instance: `az vm list-sizes --location 'northeurope' -o table`.
 - Your Azure subscription quota limits impact the ability to deploy ghaf-infra, as such, you might need to increase the vCPU quotas for your subscription via the Azure web portal. See more information at https://learn.microsoft.com/en-us/azure/quotas/quotas-overview. You can check your quota usage from the Azure web portal or using `az vm list-usage`, for instance: `az vm list-usage --location "northeurope" -o table`.
-- Currently, the terraform state is always deployed to the default Azure location, even if your ghaf-infra instance other resources (i.e. [persistent data](./persistent/main.tf), [root module resources](main.tf)) are deployed to a different location.
 
-Following shows an example of deploying ghaf-infra to Azure location UAE North:
+Following shows an example of deploying ghaf-infra to Azure location SWE Central:
 
 ```bash
-# Initialize terraform state and persistent data, using UAE North as an example location:
-$ ./terraform-init.sh -l uaenorth
+# Initialize terraform state and persistent data, using SWE Central as an example location:
+$ ./terraform-init.sh -l swedencentral
 
-# Switch to (and optionally create) a workspace 'devuaen'
-$ terraform workspace new devuaen || terraform workspace select devuaen
+# Switch to (and optionally create) a workspace 'devswec'
+$ terraform workspace new devswec || terraform workspace select devswec
 
 # Optionally, run Terraform plan:
 # (Variable 'envtype' overrides the default environment type)
-$ terraform plan -var="location=uaenorth" -var="envtype=dev"
+$ terraform plan -var="envtype=dev"
 
-# Deploy with Terraform apply, specifying the location with a variable:
-$ terraform apply -var="location=uaenorth" -var="envtype=dev" -auto-approve
-```
-When destroying a ghaf-infra instance deployed to a non-default location, you need to specify the location with a variable:
-```bash
-# Destroy:
-$ terraform apply -destroy -var="location=uaenorth" -auto-approve
+# Deploy with Terraform apply:
+$ terraform apply -var="envtype=dev" -auto-approve
```

Comment:
se on hyva...

Reply:
Not sure I understand this comment, are you saying it's good?

Reply:
yeahI think  it's good we are adding  -auto-approve